### PR TITLE
Fix driver loading issue

### PR DESCRIPTION
**Pull request by ishan1729**

### CODE REVIEW

File: javasource/databaseconnector/impl/JdbcConnectionManager.java [hunk 1]
```diff
@@ -68,10 +68,10 @@ public Connection getConnection(final String jdbcUrl, final String userName, fin
 	private synchronized void initializeDrivers() {
 		if (!hasDriversInitialized) {
 			ServiceLoader<Driver> loader = ServiceLoader.load(Driver.class);
+			Stream<String> driverNames = StreamSupport.stream(loader.spliterator(), false)
+					.map(a -> a.getClass().getName());
+			String logMessage = driverNames.collect(Collectors.joining(", ", "Found JDBC Drivers: ", ""));
 			if (logNode.isTraceEnabled()) {
```
Comment (basdebakker) [inline javasource/databaseconnector/impl/JdbcConnectionManager.java:74]: This `if` is no longer useful.

Comment (ishan1729) [reply javasource/databaseconnector/impl/JdbcConnectionManager.java:74]: Interesting. I am bit confused now as to why we needed this `if` condition in the first place. I thought we needed it for the `log.trace` statement. 🤔  
So, it was for the log message then? Do you know why was that useful before?

Comment (ishan1729) [reply javasource/databaseconnector/impl/JdbcConnectionManager.java:82]: I have made some changes which makes the registration of drivers explicit. I guess the `if` can stay now?

File: javasource/databaseconnector/impl/JdbcConnectionManager.java [hunk 2]
```diff
@@ -61,19 +63,28 @@ public Connection getConnection(final String jdbcUrl, final String userName, fin
...
-	 * the drivers are put into the project.jar on deployment. Iterating through the
-	 * drivers found by ServiceLoader.load(Driver.class) will force them to load and
-	 * register to the DriverManager.
+	 * the drivers are put into the project.jar on deployment. Hence, we explicitly
+	 * register the drivers.
 	 */
 	private synchronized void initializeDrivers() {
 		if (!hasDriversInitialized) {
 			ServiceLoader<Driver> loader = ServiceLoader.load(Driver.class);
+			List<Driver> drivers = StreamSupport.stream(loader.spliterator(), false).collect(Collectors.toList());
```
Comment (basdebakker) [inline javasource/databaseconnector/impl/JdbcConnectionManager.java:72]: Why not use `loader.stream().collect(...)`?

File: javasource/databaseconnector/impl/JdbcConnectionManager.java (hunk 1 above)
Comment (basdebakker) [reply javasource/databaseconnector/impl/JdbcConnectionManager.java:82]: Yes.

File: javasource/databaseconnector/impl/JdbcConnectionManager.java (hunk 2 above)
Comment (basdebakker) [reply javasource/databaseconnector/impl/JdbcConnectionManager.java:72]: Another good point.

**Comment by dahfjkg** [comment]
Validated, and it works fine. Let's update database drivers.

**Pull request merged**